src/ResourceManager/Compute/Commands.Compute/help/Get-AzureRmDisk.md

```diff
@@ -82,7 +82,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## OUTPUTS
 
-### System.Object
+Microsoft.Azure.Commands.Compute.Automation.Models.PSDisk
```

Comment:
@hyonholee can you make sure that these changed outputs are headers?

`Microsoft.Azure.Commands.Compute.Automation.Models.PSDisk` 

becomes

`### Microsoft.Azure.Commands.Compute.Automation.Models.PSDisk`